awxkit/awxkit/api/pages/schedules.py

```diff
@@ -25,7 +26,7 @@ def silent_delete(self):
             pass
 
 
-page.register_page([resources.schedule, resources.related_schedule], Schedule)
+page.register_page([resources.schedule, resources.related_schedule, (resources.schedule, 'post')], Schedule)
```

Comment:
I think you intended `(resources.schedules, 'post')` here.  You are trying to say, "when I do a post to the schedules list page, the resulting object is a schedule".

Reply:
What you currently have is like saying, "when I post changes to a schedule detail page, the resulting object is a schedule", and is already implied by the first `resources.schedule` entry in the list.

Reply:
This was left over from my debugging and I did not mean to commit it.  Thanks for catching it.